docs/sources/static/set-up/install-agent-on-windows.md

```diff
@@ -1,48 +1,57 @@
 
-    Grafana Agent is installed into the default directory `C:\Program Files\Grafana Agent`.
-    The [windows_exporter integration](https://grafana.com/docs/agent/latest/static/configuration/integrations/windows-exporter-config)
-    can be enabled with all default windows_exporter options.
+## Standard installation
 
-1. Check you can access `http://localhost:12345/-/healthy` and `http://localhost:12345/agent/api/v1/metrics/targets`.
+1. Unzip the downloaded file.
+1. Double-click on `grafana-agent-installer.exe` to install Grafana Agent.
 
+   Grafana Agent is installed into the default directory `C:\Program Files\Grafana Agent`.
+   The [windows_exporter integration](https://grafana.com/docs/agent/latest/static/configuration/integrations/windows-exporter-config) can be enabled with all default windows_exporter options.
 
+## Silent installation
 
-1. (Optional): You can adjust `C:\Program Files\Grafana Agent\agent-config.yaml` to meet your specific needs. After changing the configuration file, restart the Grafana Agent service to load changes to the configuration.
+1. Unzip the downloaded file.
+1. Run the following command in PowerShell or Command Prompt:
+
+   ```shell
+   <path_to_installer>/grafana-agent-flow-installer.exe /S
+   ```
+## Silent installation with `remote_write`
 
-   Existing configuration files are kept when re-installing or upgrading the Grafana Agent.
+If you are using `remote_write` you must enable Windows Exporter and set the global remote_write configuration. 
 
-## Silent Installation
+1. Unzip the downloaded file.
+1. Run the following command in PowerShell or Command Prompt:
 
-You can install Grafana Agent using silent installation as follows.
+   ```shell
+   <path_to_installer>/grafana-agent-flow-installer.exe /S /EnableExporter true /Username <username> /Password <password> /Url "http://example.com"
+   ```
 
-1. Enter the following in your command line.
-   `grafana-agent-installer.exe /S /EnableExporter true /Username xyz /Password password /Url "http://example.com" `
+   If you are using Powershell, make sure you use triple quotes `"""http://example.com"""` around the URL parameter.
 
-1. Set EnableExporter to enable Windows Exporter. The default is `false`.
-1. Enter a Username, Password, and URL to set the global remote_write configuration.
+## Verify the installation
 
-  You do not need to set username, password, and URL if you are not using remote_write.
-  If you are using powershell, use triple quotes `"""http://example.com"""` around the URL parameter around the url parameter.
+1. Makee sure you can access `http://localhost:12345/-/healthy` and `http://localhost:12345/agent/api/v1/metrics/targets`.
+1. (Optional): You can adjust `C:\Program Files\Grafana Agent\agent-config.yaml` to meet your specific needs. After changing the configuration file, restart the Grafana Agent service to load changes to the configuration.
```

Comment:
It looks like the documentation style guide on [procedures](https://developers.google.com/style/procedures#optional-steps) recommends not wrapping the optional indicator in parenthesis:

```suggestion
1. Optional: You can adjust `C:\Program Files\Grafana Agent\agent-config.yaml` to meet your specific needs. After changing the configuration file, restart the Grafana Agent service to load changes to the configuration.
```